sass/components/_services-section.scss

```diff
@@ -0,0 +1,106 @@
+
+/*---services section css---*/
+
+%services {
+  margin-bottom: 4em;
+  & h2 {
+    line-height: 3em;
+  }
+}
+
+%bkgrnd-image {
+  display: block;
+  height: 6em;
+  content: '';
+  background-size: contain;
+  background-position: center;
+}
+
+.service-intro {
+  margin: auto 13%;
+}
+
+.column-two {
+  -webkit-column-count: 2;
+     -moz-column-count: 2;
+          column-count: 2;
+  margin: 2em;
+  @media only screen and (max-width: 600px) {
+    -webkit-column-count: 1;
+     -moz-column-count: 1;
+          column-count: 1;
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+  }
+}
+
+.column {
+  min-width: 33%;
+  margin: 0 10%;
+}
+
+.column-left {
+  margin-left: 23%;
+  margin-right: 1em;
+  font-size: .9em;
+  @media only screen and (max-width: 600px) {
+    margin: 2em 4em 2em 4em;
+  }
+}
+
+.column-right {
+  margin-right: 23%;
+  margin-left: 1em;
+  font-size: .9em;
+  @media only screen and (max-width: 600px) {
+    margin: 0 4em 2em 4em;
+  }
+}
+
+.services-container {
+  width: 100%;
+  @include flexbox;
+  padding: 0 5%;
+}
+
+.services-item {
+  width: 20em;
+  padding: 0 2%;
+  text-align: center;
+  min-width: 20em;
+}
+
+.ux-design {
+  margin-bottom: 4em;
+  h2 {
+    line-height: 3em;
+    &:after {
+      @extend %bkgrnd-image;
+      background: url("./../imgs/brush.gif") 50% 0 no-repeat;
```

Comment:
I would have preferred to see an SVG or at least a higher resolution bitmap. As delivered, this does not look good on retina display.